### DISCUSSION TITLE
Confusing demo for path specification

### DISCUSSION THREAD

**Discussion by rojakcoder** [PaloAltoNetworks/docusaurus-openapi-docs#511] [Q&A]
In the demo site (https://docusaurus-openapi.tryingpan.dev/petstore/add-pet), the API documentation resides in the path `/petstore/add-pet`.

I can't tell from the [demo configuration file](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/demo/docusaurus.config.js#L225:L237) how this path is specified:

```js
petstore: {
  specPath: "examples/petstore.yaml",
  proxy: "https://cors.pan.dev",
  outputDir: "docs/petstore",
  sidebarOptions: {
    groupPathsBy: "tag",
    categoryLinkSource: "tag",
  },
  template: "api.mustache", // Customize API MDX with mustache template
  downloadUrl:
    "https://raw.githubusercontent.com/PaloAltoNetworks/docusaurus-openapi-docs/main/demo/examples/petstore.yaml",
  hideSendButton: true,
},
```

And in this demo configuration, the `outputDir` specifies the `docs` directory which is used by the [@docusaurus/plugin-content-docs](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs) plugin (`outputDir: "docs/petstore"`). Does this imply that the generated files need to be in the same folder as the `plugin-content-docs` plugin?

However, in the [documentation](https://docusaurus-openapi.tryingpan.dev/#configuring-the-plugin), the example shows an output that is `outputDir: "api/petstore"`

What is the desirable ouput directory for the generated API docs?

**Comment by sserrata** [comment]
Hi @rojakcoder, the demo implements Docusaurus "docs-only mode" which is which sets the base path to `/` instead of the typical `/docs`: https://docusaurus.io/docs/docs-introduction#docs-only-mode

> Does this imply that the generated files need to be in the same folder as the plugin-content-docs plugin?

Although `plugin-content-docs` plugin is required to render the generated files, it's totally up to you to decide how you want to organize content. For example, you could choose to have all API and non-API docs under the same plugin or preset, or you can choose to separate them, each with their own `plugin-content-docs` plugin or preset. The choice of `outputDir` is also entirely up to you.
